swaap/swaap/Models/Connection Model/Contact.swift

```diff
@@ -45,8 +49,8 @@ struct ContactContainer: Decodable {
 		let pendingConnections = allConnections.filter { $0.status == .pending }
 		self.sentConnections = sentConnections.compactMap { $0.sender }
 		self.receivedConnections = receivedConnections.compactMap { $0.receiver }
-		self.connections = connections.compactMap { $0.sender ?? $0.receiver }
-		self.pendingReceivedConnections = pendingConnections.compactMap { $0.sender }
-		self.pendingSentConnections = pendingConnections.compactMap { $0.receiver }
+		self.connections = connections
```

Comment:
this is somewhat repetitive - look at directly setting self.connections to where `connections` is created

Reply:
fixed